recipe/meta.yaml

```diff
@@ -9,12 +9,12 @@ package:
 
 source:
   git_url: https://github.com/tensorflow/probability.git
-  git_rev: v{{ version }}
+  git_rev: v{{ version }}-rc0
   patches:
     - 0002-revert-README-addition.patch
 
 build:
-  number: 1
+  number: 2
```

Comment:
Build number should be 1.

Reply:
right, since the main version number updated, we keep the build number at 1.